juriscraper/pacer/email.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, court_id):
         self.court_id = court_id
         self.content_type = None
         self.appellate = None
-        self.content_encoding = "utf-8"
+        self.claim = None
```

Comment:
Would you consider using `self.ERROR_STRINGS` for this instead? I think it could simplify a bit? It's used in other PACER reports to identify errant input, generally.